src/environments/environment.prod.ts

```diff
@@ -7,14 +7,14 @@ export const environment: Config = {
     order: 'https://.execute-api.eu-west-1.amazonaws.com/dev',
     import: 'https://eq8dcnldy0.execute-api.eu-west-1.amazonaws.com/dev',
     bff: 'https://8lqt13zye6.execute-api.eu-west-1.amazonaws.com',
-    cart: 'https://.execute-api.eu-west-1.amazonaws.com/dev',
+    cart: 'http://localhost:8080/api',
```

Comment:
localhost???

Reply:
omg!!  Sorry, try to test locally and forget to use remote API. Fixed